Mjml.Net/MjmlRenderer.cs

```diff
@@ -14,10 +15,27 @@ namespace Mjml.Net
     /// </summary>
     public sealed class MjmlRenderer : IMjmlRenderer
     {
-        private static readonly Regex AttributeRegex = new Regex(@"\s*(?<Name>[a-z0-9]*)=""(?<Value>.*)""([\s]|>)", RegexOptions.Compiled | RegexOptions.ExplicitCapture);
+        private readonly Regex attributeRegex = new Regex(@"\s*(?<Name>[a-z0-9]*)=""(?<Value>.*)""([\s]|>)", RegexOptions.Compiled | RegexOptions.ExplicitCapture);
```

Comment:
This is vulnerable to ReDoS attacks. You can try it here:
https://devina.io/redos-checker

We should do security review at the end on all regexes possibly.